### PR TITLE
Add license of shred, cc #5500

### PR DESCRIPTION
**Pull request by NiniHsu**
https://github.com/pandastrike/shred/blob/master/LICENSE

### CODE REVIEW

**Comment by NiniHsu** [comment]
@kennynaoh 
Please help me check this PR.
Thank you!

**Comment by kennynaoh** [comment]
@NiniHsu 
The reference you commit is a git repo url.
Please fix it.
Thanks!

**Comment by NiniHsu** [comment]
@kennynaoh 
I fixed the url.
Please check it.
Thank you!

File: ajax/libs/shred/package.json [hunk 1]
```diff
@@ -22,5 +22,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/spire-io/shred"
-  }
+  },
+  "license": "ISC"
 }
+
```
Comment (kennynaoh) [inline ajax/libs/shred/package.json]: Please remove the blank line, thanks!

**Comment by NiniHsu** [comment]
@kennynaoh 
I removed it.
Thank you!

**Comment by kennynaoh** [comment]
@NiniHsu please use more meaningful branch name next time. Thanks!

**Comment by kennynaoh** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Thanks for your contribution.

**Pull request merged**